gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilExtension.kt

```diff
@@ -50,6 +50,13 @@ public abstract class AnvilExtension @Inject constructor(objects: ObjectFactory)
   public val syncGeneratedSources: Property<Boolean> = objects.property(Boolean::class.java)
     .convention(false)
 
+  /**
+   * Add the `annotations-optional` artifact as a dependency to make `SingleIn` and `ForScope` 
+   * available to use.
```

Comment:
Minor re-wording suggestion:
```suggestion
   * Add the `annotations-optional` artifact as a dependency to make annotations like `@SingleIn` and `@ForScope` 
   * available to use. These are annotations that are not strictly required but which we've found to be helpful with managing larger dependency graphs.
```